Jint/Native/Object/ObjectConstructor.cs

```diff
@@ -298,14 +301,14 @@ public JsValue Freeze(JsValue thisObject, JsValue[] arguments)
                 {
                     if (desc.Writable)
                     {
-                        var mutable = desc as PropertyDescriptor ?? new PropertyDescriptor(desc);
```

Comment:
Why aren't we cloning it anymore? Are these immutable?

Reply:
R# showed that this will never be evaluated, every property descriptor is a PropertyDescriptor..